web/app/editor/DateEditor.jsx

```diff
@@ -49,10 +49,23 @@ DateEditor.validateModel = (model) => {
 }
 
 const isOptional = (model) =>
-  model.optional && !isPerusopetuksenVuosiluokanSuorituksenAlkamispäivä(model)
+  model.optional &&
```

Comment:
Jos näitä samoja tarkastuksia tehdään tulevaisuudessa lisää erityyppisille suorituksille, kannattaa toiminnallisuus splitata omaan funktioonsa, jotta vältytään copypastelta.